react/features/settings/components/native/SettingsView.tsx

```diff
@@ -590,17 +736,19 @@ class SettingsView extends AbstractSettingsView<Props, State> {
  * @param {Object} state - The Redux state.
  * @returns {Props}
  */
-function _mapStateToProps(state) {
+function _mapStateToProps(state: any) {
     const localParticipant = getLocalParticipant(state);
     const localParticipantId = localParticipant?.id;
     const avatarLabel = localParticipant && getParticipantDisplayName(state, localParticipantId);
```

Comment:
Shouldn't this avatarLabel be the display name from base/settings?

Reply:
We can rely on that, yes. I'll make the change.